src/node/consumer/src/lib/models/configuration/cache_config.ts

```diff
@@ -0,0 +1,15 @@
+import {BooleanHelpers} from "../../helpers/boolean_helpers";
+
+export class CacheConfig {
```

Comment:
Like my comments above, and from a functional programming POV, this doesn't *need* to be a class and could be a plain and simple JS object instead. The `MONGO_URI` getter could then be converted to a utility function which accepts the config as a parameter. I'll just leave this here as I'm not sure what direction is favored.

Reply:
Good call
I'll convert to a constant exported object and move the get function to the cache service.
I think its important to at least keep these in their own model files, as the config file was getting pretty busy, and separately these are easier to maintain